src/components/wallet/methods/depositAsset.ts

```diff
@@ -163,8 +163,9 @@ export default async function depositAsset(
     if (!interactiveResponse.url) {
       throw 'No URL Returned from POST /transactions/deposit/interactive'
     }
-
-    const urlBuilder = new URL(interactiveResponse.url)
+    const urlBuilder = new URL(
+      interactiveResponse.url.replace(/^http:\/\//i, 'https://')
+    )
```

Comment:
💡 Instead of breaking out the big guns regex you can change the protocol directly on the `urlBuilder` object.
```suggestion
    const urlBuilder = new URL(interactiveResponse.url)
    urlBuilder.protocol = 'https'
```

Reply:
Oh wow thats a lot easier 😅 
Thanks for the suggestion.